dbscripts/xml/upgrade.xml

```diff
@@ -190,6 +195,8 @@
 		<code function="installEmailTemplate" key="EDITOR_DECISION_CANCEL_REVIEW_ROUND" locales="en_US" />
 		<code function="installEmailTemplate" key="REVIEW_RESEND_REQUEST" locales="en_US" />
 		<code function="installEmailTemplate" key="DISCUSSION_NOTIFICATION" locales="en_US" />
+		<code function="installEmailTemplate" key="SUBMISSION_SAVED_FOR_LATER" locales="en_US" />
+		<code function="installEmailTemplate" key="SUBMISSION_NEEDS_EDITOR" locales="en_US" />
```

Comment:
Since https://github.com/pkp/pkp-lib/issues/7673, it should no longer be necessary (or desirable!) to specify a list of locales for these.